src/components/filter/filter-store.tsx

```diff
@@ -156,67 +100,24 @@ export const useFilterStore = create<FilterState>()((set, get) => ({
 			[showWaterNeedTrees ? "true" : "false"],
 		);
 		useUrlState.getState().setSearchParams(updatedSearchParams);
-
-		if (showWaterNeedTrees) {
-			set({
-				treeAgeIntervals: get().treeAgeIntervals.map((int) => {
-					return {
-						...int,
-						enabled: int.identifier === TreeAgeIntervalIdentifier.Medium,
-					};
-				}),
-			});
-		} else {
-			set({ treeAgeIntervals: initialTreeAgeIntervals });
-		}
-
-		const updatedIntervalIdentifiers = get()
-			.treeAgeIntervals.filter((int) => int.enabled)
-			.map((int) => {
-				return int.identifier.toString();
-			});
-		const updatedTreeIntervalSearchParams = replaceUrlSearchParam(
-			new URL(window.location.href),
-			treeAgeUrlKey,
-			updatedIntervalIdentifiers,
-		);
-		useUrlState.getState().setSearchParams(updatedTreeIntervalSearchParams);
 	},
 
-	toggleTreeAgeInterval: (interval) => {
-		const updatedIntervals = get().treeAgeIntervals.map((int) => {
-			if (int.identifier === interval.identifier) {
-				return { ...int, enabled: !int.enabled };
-			}
-			return int;
-		});
-		set({ treeAgeIntervals: updatedIntervals });
+	setTreeAgeRange: (min, max) => {
+		set({ treeAgeRange: { min, max } });
 
-		const updatedIntervalIdentifiers = updatedIntervals
-			.filter((int) => int.enabled)
-			.map((int) => {
-				return int.identifier.toString();
-			});
+		const updatedRangeIdentifiers = [
+			get().treeAgeRange.min.toString() +
+				"-" +
+				get().treeAgeRange.max.toString(),
+		];
```

Comment:
no sure if we should use this format. I think it would be clearer to use two different query params, one for min and one for max

Reply:
also, if you set a specific range, then refresh, the range gets lost. see my comment regarding
> we still need something to read the state of the URL treeRange query param on initial page view

further above